components/Species.jsx

```diff
@@ -52,7 +52,7 @@ export default Species = ({ navigation }) => {
             <TouchableOpacity
               onPress={() => {
                 navigation.navigate("Bird", bird);
-                // console.log(bird);
+                navigation.navigate()
```

Comment:
Is this a typo? It throws an error saying it expects an object. When I remove it it works fine

Reply:
Yeah that shouldn't have been there so I've removed it!